### ISSUE TITLE
Java 9 upwards compatibility

### ISSUE THREAD

**Issue by janniclas** [delphi-hub/delphi-webapp#1]
The current implementation is based on features only supported in Java 8 and below.

The easy workarounds for this, like using the --add-module flag (https://stackoverflow.com/questions/42966880/java-lang-noclassdeffounderror-javax-xml-bind-datatypeconverter/42971683), is discouraged by oracle since it will be most likely removed in java 11 (I believe it might already have been removed in Java 10, since I didn't manage to apply the workaround, 
 but this needs further investigation). 
In order to keep the project up to date I suggest to timely investigate the needed steps to support newer java versions (https://docs.oracle.com/javase/9/migrate/toc.htm#JSMIG-GUID-7744EF96-5899-4FB2-B34E-86D49B2E89B6).

For reference I include he following screenshot showing the error produced when using Java 9 or newer. 

![image](https://user-images.githubusercontent.com/8062521/38768258-15f9245c-3ff1-11e8-806a-ab30d4872ec3.png)

**Comment by bhermann** [comment]
On first look, this seems to be an issue of Akka which is used by Play. We might have to wait for the proper updates or even create issues in the dependent projects.

**Comment by janniclas** [comment]
@bhermann yeah maybe, there was a quite long stack trace following this first exception. I will update the issue after I have further investigated it.

**Comment by janniclas** [comment]
The error was actually mentioned in the playframework repository (https://github.com/playframework/playframework/issues/7879) and the workaround to extend the dependencies in build.sbt with the following code worked for me with Java 10: 

```
libraryDependencies += "javax.xml.bind" % "jaxb-api" % "2.3.0" 
libraryDependencies += "javax.annotation" % "javax.annotation-api" % "1.3.2" 
libraryDependencies += "javax.el" % "javax.el-api" % "3.0.0" 
libraryDependencies += "org.glassfish" % "javax.el" % "3.0.0" 
```
Since I don't have any experience with either sbt, nor the play framework yet, I will wait to create a pull request until I fully understand what this does. Until then it is possible to use Java > 8 by extending the bild.sbt with the above mentioned code.

**Comment by bhermann** [comment]
Closed with upgrade to sbt 1.3.8, Scala 2.13.1, and Play 2.8.0
